tests/RobotFramework/tests/mqtt/disconnect_from_broker.robot

```diff
@@ -0,0 +1,65 @@
+*** Settings ***
+Resource        ../../resources/common.resource
+Library         ThinEdgeIO
+Library         OperatingSystem
+Library         String
+
+Suite Setup     Setup    skip_bootstrap=True
+Suite Teardown  Get Logs
+
+Test Tags      theme:mqtt
+
+
+*** Test Cases ***
+
+Publish Connection Closed after publish and no error message 
+    [Documentation]    Tests that the connection to the MQTT broker is closed after publishing.
+    Execute Command    /setup/bootstrap.sh
+    Execute Command    tedge mqtt pub test/topic Hello
+    ${MQTT_PUB}    Execute Command    journalctl -u mosquitto -n 30
+    Should Contain    ${MQTT_PUB}    Received DISCONNECT from tedge-pub
+    Should Not Contain    ${MQTT_PUB}    Error
```

Comment:
I'm pretty sure the bootstrap.sh script will use TLS if the global default is to use TLS (which I believe is the default).

And the assertion confirms this, as if a non-TLS connection was being used, then we wouldn't get the "Error" line in the log (as @didier-wenzek confirmed this earlier in a different discussion).

```
Should Not Contain ${MQTT_PUB} Error
```

So it might just be easier to remove the "Error" assertion altogether, or force using non-TLS setting when calling the bootstrap.sh